_search-plugins/sql/functions.md

```diff
@@ -68,35 +68,80 @@ sinh | `sinh(number T) -> double` | `SELECT sinh(0.5) FROM my-index LIMIT 1`
 tan | `tan(number T) -> double` | `SELECT tan(0.5) FROM my-index LIMIT 1`
 
 ## Date and time
+Functions marked with * have future PPL implementation on roadmap.
```

Comment:
```suggestion
Functions marked with * are only available in SQL. 
```

Reply:
Addressed in https://github.com/opensearch-project/documentation-website/pull/3908/commits/e033e7fa5264764d0226feee54fb4d99acc9afbd